protocol/src/types/metadata.rs

```diff
@@ -657,21 +652,18 @@ impl Serializable for ParticleData {
 
 #[derive(Debug)]
 pub struct VillagerData {
-    villager_type: protocol::VarInt,
-    profession: protocol::VarInt,
-    level: protocol::VarInt,
+    // TODO: use these
```

Comment:
Same here

Reply:
This code was not used. Clippy linter now complaints about dead code. This is intentional. If anyone will implement this, he can uncomment the relevant code. And it is easy to fing unimplemented stuff because it is marked with a TODO.

Reply:
That's okay, but these TODOs and the commented out code is all over the place and there doesn't seem to be a clear structure for commented out code and sometimes there isn't even a comment regarding what the TODO actually refers to